pkg/core/statesync/neotest_test.go

```diff
@@ -419,8 +467,12 @@ func TestStateSyncModule_RestoreBasicChain(t *testing.T) {
 			}
 			require.Error(t, module.AddBlock(b))
 		})
+		start := uint32(stateSyncPoint - maxTraceable + 1)
+		if enableStorageSync {
+			start = 1
```

Comment:
It's an unexpected behaviour and we should fix the implementation instead of breaking unit-tests. It's fixed in https://github.com/nspcc-dev/neo-go/pull/3933/commits/edebb29818c33ea1f72d927ada159bdc10bdccf6, rebase onto master.